.github/dependabot.yml

```diff
@@ -60,3 +60,11 @@ updates:
     labels:
       - "dependencies"
       - "area:actions"
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "docker:"
+    labels:
+      - "dependencies"
```

Comment:
I checked the documentation and this wont work. `docker` ecosystem only check the tag of k8s chart manifest.
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

Reply:
oops sorry should have read this more carefully